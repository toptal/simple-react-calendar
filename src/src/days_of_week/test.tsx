import {shallow} from 'enzyme'
import * as Luxon from 'luxon'
import React from 'react'

import DaysOfWeek, {DaysOfWeekProps} from '.'

describe('DaysOfWeek', () => {
  let wrapper, props

  beforeEach(() => {
    props = getProps()
    wrapper = shallow(<DaysOfWeek {...props} />)
  })

  describe('#render', () => {
    it('renders <DaysOfWeek />', () => {
      expect(wrapper).toMatchSnapshot()
    })
  })
})

const getProps = (overrides: Partial<DaysOfWeekProps> = {}): DaysOfWeekProps => ({
  blockClassName: 'simple-react-calendar',
  daysOfWeek: Luxon.Info.weekdays('short'),
  ...overrides,
})
