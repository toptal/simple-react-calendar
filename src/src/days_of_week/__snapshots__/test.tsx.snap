// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DaysOfWeek #render renders <DaysOfWeek /> 1`] = `
<ol
  className="simple-react-calendar-days_of_week"
>
  <li
    className="simple-react-calendar-days_of_week_day"
    key="Mon"
  >
    Mon
  </li>
  <li
    className="simple-react-calendar-days_of_week_day"
    key="Tue"
  >
    Tue
  </li>
  <li
    className="simple-react-calendar-days_of_week_day"
    key="Wed"
  >
    Wed
  </li>
  <li
    className="simple-react-calendar-days_of_week_day"
    key="Thu"
  >
    Thu
  </li>
  <li
    className="simple-react-calendar-days_of_week_day"
    key="Fri"
  >
    Fri
  </li>
  <li
    className="simple-react-calendar-days_of_week_day is-weekend"
    key="Sat"
  >
    Sat
  </li>
  <li
    className="simple-react-calendar-days_of_week_day is-weekend"
    key="Sun"
  >
    Sun
  </li>
</ol>
`;
